modules/getting-started-web-console-granting-permissions.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="getting-started-web-console-granting-permissions_{context}"]
+= Granting view permissions
+
+{product-title} automatically creates a few special service accounts in every project. The default service account takes responsibility for running the Pods. {product-title} uses and injects this service account into every Pod that launches.
```

Comment:
Same comments in this intro as the other granting perms module (lower casing Pod/other objs)